### DISCUSSION TITLE
Error: java.lang.NullPointerException: Cannot invoke "net.fabricmc.fabric.api.renderer.v1.Renderer.meshBuilder()" because the return value of "net.fabricmc.fabric.api.renderer.v1.RendererAccess.getRenderer()" is null

### DISCUSSION THREAD

**Discussion by DrevCronsole** [FabricMC/fabric-api#2294] [Player support]
happened when i was placing a dirt block to gt to iron then it crashed with this error

client crash log:
[crash-2022-06-08_11.13.00-client.txt](https://github.com/FabricMC/fabric/files/8862406/crash-2022-06-08_11.13.00-client.txt)

**Comment by warjort** [comment]
Same error as #2295?
